src/doc/src/reference/specifying-dependencies.md

```diff
@@ -183,53 +184,95 @@ you need to specify is the location of the repository with the `git` key:
 regex = { git = "https://github.com/rust-lang/regex.git" }
 ```
 
-Cargo will fetch the `git` repository at this location then look for a
-`Cargo.toml` for the requested crate anywhere inside the `git` repository
-(not necessarily at the root --- for example, specifying a member crate name
-of a workspace and setting `git` to the repository containing the workspace).
+Cargo fetches the `git` repository at that location, then looks for
+`Cargo.toml` file for the requested crate anywhere inside the `git` repository.  
```

Comment:
@weihanglo , @ehuss  can you point me at the code that does that? It is not clear if Cargo will only look at the list of member crates or will traverse the entire tree.